### ISSUE TITLE
Crashes Matlab 2023b 

### ISSUE THREAD

**Issue by ivandemec** [xiangruili/uiFileDnD#4]
When you DnD multiple files to a ListBox, it crashes Matlab every time, w/o any error reported. When you DnD just one file it works fine from time to time, but it also crashes.
With 2023a it worked more or less fine, with occasionally crashing. Now it's completely unusable with the new release.

**Comment by xiangruili** [comment]
The crash issue has been there from the beginning. It seems it becomes worse with late versions of Matlab. 
I tried to add a video frame of wait in ondragover(), and hopefully it helps. I don't have the latest Matlab. If you find the fix work, please let me know. Thanks.

**Comment by ivandemec** [comment]
Ok so it works, seems so.
When I implemented your change, it first crashed regularly like before. Then, I tried to do it in another app, on a label component, and it worked fine. Which lead me to a thought that it might be a problem with the ListBox component. But, when I returned to my app it somehow started working again... strange 🙃 And now it works seamlessly, like expected to. 
I'll have quite some people testing this app in the months to come. Will let you know if the problem persists. But, for now I'd close this issue.
